prow/crier/reporters/gerrit/reporter.go

```diff
@@ -106,7 +106,14 @@ func NewReporter(cfg config.Getter, cookiefilePath string, projects map[string][
 				time.Sleep(time.Second)
 				continue
 			}
+			// Updates clients based on global gerrit config.
 			gc.UpdateClients(orgReposConfig.AllRepos())
```

Comment:
It doesn't look like we respect the global org repo config immediately when `NewReporter` is called, that happens asynchronously.  If we follow the same pattern used by the gerrit adapter (maybe even use the same code) we won't risk transient errors on start up.

Reply:
good point, updated the code.

> use the same code

Technically this could be done by adding this logic in gerrit Client, however it might be an overkill for gerrit Client to understand prow config, so duplicated code instead, which is not too much code anyways